### PR TITLE
#218 removed unneccessary content in subtitle

### PR DESCRIPTION
**Pull request by MaiThiD**

### CODE REVIEW

**Review by FiedlerJan — COMMENTED** [review]
I'm not sure, but didn't we say that the 'post_struct' may be accessed for this display? That would make the code in the frontend easier and there would be no unnecessary commas. @BernhardHumm

**Comment by BernhardHumm** [comment]
I don’t understand what this issue is about.

 

 

 

 

Von: JanF <notifications@github.com> 
Gesendet: Donnerstag, 10. Dezember 2020 23:54
An: hochschule-darmstadt/einander-helfen <einander-helfen@noreply.github.com>
Cc: Humm, Bernhard, Prof. Dr. <bernhard.humm@h-da.de>; Mention <mention@noreply.github.com>
Betreff: Re: [hochschule-darmstadt/einander-helfen] #218 removed unneccessary content in subtitle (#233)

 

@FiedlerJan commented on this pull request.

I'm not sure, but didn't we say that the 'post_struct' may be accessed for this display? That would make the code in the frontend easier and there would be no unnecessary commas. @BernhardHumm <https://github.com/BernhardHumm>

**Comment by FiedlerJan** [comment]
> I don’t understand what this issue is about.

Please take a look at #218.

**Review by DianaPankratz — APPROVED** [review]

**Comment by BernhardHumm** [comment]
Yes, use the structured information from location (which should only contain content from the original post). Use the following attributes if existing:

Zip code

City

Country

 

For national posts, it will presumably be City or Zip code + City, for international posts presumably City + Country.

 

 

Von: Humm, Bernhard, Prof. Dr. 
Gesendet: Freitag, 11. Dezember 2020 13:16
An: 'hochschule-darmstadt/einander-helfen' <reply@reply.github.com>
Betreff: AW: [hochschule-darmstadt/einander-helfen] #218 removed unneccessary content in subtitle (#233)

 

I don’t understand what this issue is about.

 

 

 

 

Von: JanF <notifications@github.com <mailto:notifications@github.com> > 
Gesendet: Donnerstag, 10. Dezember 2020 23:54
An: hochschule-darmstadt/einander-helfen <einander-helfen@noreply.github.com <mailto:einander-helfen@noreply.github.com> >
Cc: Humm, Bernhard, Prof. Dr. <bernhard.humm@h-da.de <mailto:bernhard.humm@h-da.de> >; Mention <mention@noreply.github.com <mailto:mention@noreply.github.com> >
Betreff: Re: [hochschule-darmstadt/einander-helfen] #218 removed unneccessary content in subtitle (#233)

 

@FiedlerJan commented on this pull request.

I'm not sure, but didn't we say that the 'post_struct' may be accessed for this display? That would make the code in the frontend easier and there would be no unnecessary commas. @BernhardHumm <https://github.com/BernhardHumm>

File: app/src/views/ResultPage.vue [hunk 1]
```diff
@@ -156,8 +156,12 @@
                       {{post.title}}
                     </v-list-item-title>
                     <v-list-item-subtitle :set="distance = postDistance(post)">
-                      <strong>{{post.location}} <em v-if="distance">(in {{distance}})</em></strong> &mdash;
-                       <span v-html="post.task"/>
+                      <strong>
+                        <span v-if="post.post_struct.location.country === 'Deutschland'">{{post.post_struct.location.zipcode}}</span>
+                        <span> {{post.post_struct.location.city}}</span>
```
Comment (FiedlerJan) [inline app/src/views/ResultPage.vue:161]: Please add the post_struct with country, city and zipcode to the post.ts model.

**Review by FiedlerJan — CHANGES_REQUESTED** [review]

**Review by FiedlerJan — APPROVED** [review]

**Review by LeDrieHda — APPROVED** [review]

**Pull request merged**